azkaban-common/src/main/java/azkaban/imagemgmt/rampup/ImageRampupManger.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package azkaban.imagemgmt.rampup;
+
+import azkaban.imagemgmt.exeception.ImageMgmtException;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * The interface exposes method for getting version for each image types based on rampup. Flow
+ * container will invoke the exposed methods to get version for the image types
```

Comment:
Nit: full stop in the end. And dispatch will invoke the exposed methods.

Reply:
Done.